activerecord/lib/active_record/store.rb

```diff
@@ -44,6 +44,17 @@ module ActiveRecord
   #     store_accessor :settings, :privileges, :servants
   #   end
   #
+  #   # Optionally specify prefix/suffix to accessor with _prefix/_suffix option
+  #   class User
+  #     store :settings, accessors: [ :two_factor_auth ], coder: JSON, _prefix: true
+  #     store_accessor :settings, :secret_question, _prefix: 'config'
```

Comment:
Should it be `_prefix` or `prefix` like in active support's `delegate`? I guess `enum` have underscored options but I'm not sure which is preferred.

Reply:
Without underscore, is the preffered way.